be/src/runtime/tablet_writer_mgr.cpp

```diff
@@ -315,6 +324,48 @@ Status TabletWriterMgr::cancel(const PTabletWriterCancelRequest& params) {
     return Status::OK;
 }
 
+Status TabletWriterMgr::start_bg_worker() {
+    _tablets_channel_clean_thread = std::thread(
+         [this] {
+             _tablets_channel_clean_thread_callback(nullptr);
+         });
+    _tablets_channel_clean_thread.detach();
+    return Status::OK;
+}
+
+void* TabletWriterMgr::_tablets_channel_clean_thread_callback(void* arg) {
```

Comment:
you can remove this function and put this logical to lambda in `start_bg_worker`.

Reply:
done